acapy_plugin_toolbox/holder/v0_1/__init__.py

```diff
@@ -118,7 +122,7 @@ async def issue_credential_event_handler(profile: Profile, event: Event):
         CredExRecord.STATE_OFFER_RECEIVED,
         CredExRecord.STATE_CREDENTIAL_RECEIVED,
     ):
-        return
+        raise IncorrectState("Record is not in the correct state.")
```

Comment:
Alternate states aren't an error condition, actually -- we just don't want to take any action on those other states. So the early return is what we want for this one :slightly_smiling_face: